.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["-Ctarget-feature=+sse4.1,+sse4.2"]
```

Comment:
These are target-specific flags and should only be applied to x86_64 builds in order to not break aarch64 builds.

```
[target.cfg(target_arch="x86_64")]
rustflags = ["-Ctarget-feature=+sse4.1,+sse4.2"]
```

Reply:
Thanks, fixed. Shouldn't smth similar be done in the Makefile?